### PR TITLE
Add motion_state action_set with min model.

### PR DESCRIPTION
**Pull request by reiterl**
Add the action set for motion_state with a minimal model.
The model is still todo later.
Add tests.

### CODE REVIEW

File: openslides_backend/models/motion_state.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    )
+    css_class = fields.CharField(description="The css class of thie motion state.")
+    # TODO restrictions: string[],
+    # TODO allow_support: boolean;
+    # TODO allow_create_poll: boolean;
+    # TODO allow_submitter_edit: boolean;
+    # TODO set_number: boolean;
+    # TODO show_state_extension_field: boolean;
+    # TODO merge_amendment_into_final: number;
+    # TODO show_recommendation_extension_field: boolean;
```
Comment (jsangmeister) [inline openslides_backend/models/motion_state.py:30]: In #101 I added `BooleanField` and `ArrayField` so you can implement these fields now. For business logic (which fields can be set by user on create/update, @FinnStutzenstein should probably weigh in again ;)

File: openslides_backend/models/motion_state.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+    collection = Collection("motion_state")
+    verbose_name = "motion_state"
+
+    id = fields.IdField(description="The id of this motion state.")
+    name = fields.RequiredCharField(description="The name of this motion state.")
+    recommendation_label = fields.CharField(
+        description="The recommendation label of thie motion state."
+    )
+    css_class = fields.CharField(description="The css class of thie motion state.")
```
Comment (jsangmeister) [inline openslides_backend/models/motion_state.py:22]: typo

Comment (reiterl) [reply openslides_backend/models/motion_state.py]: Fixed.

File: openslides_backend/models/motion_state.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/models/motion_state.py]: Merged #101 and added these fields.

**Pull request merged**